sqlalchemy_cockroachdb/base.py

```diff
@@ -148,12 +148,20 @@ def initialize(self, connection):
         # The crdb_internal.increment_feature_counter only exists in v21.1 of CRDB
         # and onwards.
         if self._is_v211plus and not self.disable_cockroachdb_telemetry:
-            version = pkg_resources.require("sqlalchemy-cockroachdb")[0].version
-            telemetry_query = (
-                "SELECT crdb_internal.increment_feature_counter"
-                + f"('SQLAlchemy {version}')"
+            adapter_version = pkg_resources.require("sqlalchemy-cockroachdb")[0].version
```

Comment:
For consistency in terminology going forward, let's call this `dialect_version`.

Reply:
fixed